admin/tabs/general/alertBanner/AlertBannerModel.ts

```diff
@@ -104,6 +108,85 @@ export class AlertBannerModel extends HoistModel {
         this.refreshAsync();
     }
 
+    loadPreset(preset: PlainObject) {
+        this.formModel.setValues({...preset, expires: null});
+    }
+
+    @action
+    addPreset() {
+        const {message, intent, iconName, enableClose} = this.formModel.values,
+            dateCreated = Date.now(),
+            createdBy = XH.getUsername();
+        this.savedPresets = sortBy(
+            [
+                ...this.savedPresets,
+                {message, intent, iconName, enableClose, dateCreated, createdBy}
+            ],
+            ['intent', 'message']
+        );
+        this.savePresetsAsync();
+    }
+
+    @action
+    async removePreset(preset: PlainObject) {
+        const confirmRemove = await XH.confirm({
+            message: fragment(
+                p(
+                    'You are removing a preset banner configuration. A user may have saved this, see the preset subtext for the creator.'
+                ),
+                p(
+                    'Choose below whether you would like to remove this configuration from the admin list of saved presets.'
+                )
+            ),
+            cancelProps: {
+                text: 'Cancel',
+                outlined: true,
+                autoFocus: false
+            },
+            confirmProps: {
+                text: 'Remove',
+                intent: 'danger',
+                outlined: true,
+                autoFocus: false
+            }
+        });
+        if (confirmRemove) {
+            this.savedPresets = without(this.savedPresets, preset);
+            this.savePresetsAsync();
+        }
+    }
+
+    @computed
+    get currentValuesSavedAsPreset() {
+        const {message, intent, iconName, enableClose} = this.formModel.values;
+        return this.savedPresets.some(
```

Comment:
here is a cool place where you can use a cool bit of  lodash:  `_(this.savedPresets).some({message, iconName, intent, enableClose})`